javascript/index.html

```diff
@@ -111,7 +111,9 @@ <h4>
             <li>
                 <a href="./answers/q5.html"><h4>What are escape Sequences?</h4></a>
             </li>
-
+            <li>
+                <a href="./answers/anonymous.html"><h4>How to define an anonymous function?</h4></a>
```

Comment:
name of answer file is in the wrong format, correct it

Reply:
done!